fish_prompt.fish

```diff
@@ -663,30 +757,418 @@ function fish_prompt -d 'bobthefish, a fish theme optimized for awesome'
   set -l __bobthefish_vagrant_unknown_glyph   '!'    # strange cases
 
   # Colors
-  set -l __bobthefish_lt_green   addc10
-  set -l __bobthefish_med_green  189303
-  set -l __bobthefish_dk_green   0c4801
-
-  set -l __bobthefish_lt_red     C99
-  set -l __bobthefish_med_red    ce000f
-  set -l __bobthefish_dk_red     600
-  set -l __bobthefish_ruby_red   af0000
-
-  set -l __bobthefish_slate_blue 255e87
-  set -l __bobthefish_med_blue   005faf
 
-  set -l __bobthefish_lt_orange  f6b117
-  set -l __bobthefish_dk_orange  3a2a03
-
-  set -l __bobthefish_dk_grey    333
-  set -l __bobthefish_med_grey   999
-  set -l __bobthefish_lt_grey    ccc
-
-  set -l __bobthefish_dk_brown   4d2600
-  set -l __bobthefish_med_brown  803F00
-  set -l __bobthefish_lt_brown   BF5E00
-
-  set -l __bobthefish_vagrant    48B4FB
+  switch "$theme_color_scheme"
+    case 'user'
+      # Do not set any variables in this section.
+
+      # If you want to create your own color scheme, set `theme_color_scheme` to
+      # `user` and define the following variables in your fish startup file. Use
+      # `__bobthefish_display_colors` to easily see what these variables are
+      # used for.
+
+      # The value for each variable is an argument to pass to `set_color -b`.
+      # See See the color schemes below for examples.
+      #
+      # - `__color_initial_segment_exit`
```

Comment:
should we pre-populate a (commented out) block of `set` commands that can just be copied and pasted into config — use the default dark colors — to make it easier for people who want to customize?